src/LibraryManager.Contracts/PredefinedErrors.cs

```diff
@@ -92,5 +94,32 @@ public static IError PathOutsideWorkingDirectory()
         public static IError VersionIsNotSupported(string version)
            => new Error("LIB009", string.Format(Text.ErrorNotSupportedVersion, version));
 
+        /// <summary>
+        /// Library is already installed by the provider.
+        /// </summary>
+        /// <param name="libraryId"></param>
+        /// <param name="providerId"></param>
+        /// <returns></returns>
+        public static IError LibraryAlreadyInstalled(string libraryId, string providerId)
+            => new Error("LIB010", string.Format(Text.ErrorLibraryAlreadyInstalled, libraryId, providerId));
+
+        /// <summary>
+        /// Library cannot be updated as updated version is already installed.
```

Comment:
If updated version is already installed, do we need an error for that? Probably just a message "Library is already up to date" or something

Reply:
This is for the following case:

User tries to do `libman update jquery@2.2.0 --to-version jquery@3.3.1` 
when the libman.json is as follows:
```json
{                                
  "version": "1.0",              
  "defaultProvider": "cdnjs",    
  "defaultDestination": "lib",   
  "libraries": [                 
    {                            
      "provider": "cdnjs",       
      "library": "jquery@2.2.0", 
      "destination": "lib"   
    },                           
    {                            
      "library": "jquery@3.3.1"  
    }                            
  ]                              
}                                
```

Reply:
I see now!